monitoring/harmony-monitor/blockchain-watchdog/reporting-server.go

```diff
@@ -165,6 +165,9 @@ func request(node string, requestBody []byte) ([]byte, []byte, error) {
 	if c != 200 {
 		return nil, requestBody, fmt.Errorf("http status code not 200, received: %d", c)
 	}
+	if len(res.Body()) == 0 {
+		return nil, requestBody, fmt.Errorf("empty reply received")
```

Comment:
Body is not idepotent , cannot call body multiple times. Do it after body called, on like line 173

Reply:
Fixed